Dockerfile

```diff
@@ -31,6 +31,9 @@ ENV KEY_PATH=config/keys/key.pem
 ENV SECRET_KEY_BASE=f7c9be19114730b947c8f7f274ea7c128e792245d049ae2c808d479f7e632817dd83009fa235326953a9ab11c70e9b0be4a8cb8657626e57ca5031a584b75295
 ENV RAILS_SERVE_STATIC_FILES=true
 
+RUN bundle exec rake webpacker:clobber
+RUN bundle exec rake webpacker:install
```

Comment:
Do we really need `webpacker:install`? Based on the documentation this seems like something that we should only have to [do once to configure Rails to use webpacker](https://github.com/rails/webpacker#installation).

Reply:
I don't think it's needed. I tried rebuilding the image without the command and everything seemed to work fine, however, I'm not sure if things would be different if I cleared my cache. I'll remove it.